EIPS/eip-4938.md

```diff
@@ -4,7 +4,7 @@ title: "eth/67: Removal of GetNodeData"
 description: "Remove GetNodeData and NodeData messages from the wire protocol"
 author: Marius van der Wijden (@MariusVanDerWijden), Felix Lange <fjl@ethereum.org>, Gary Rong <garyrong@ethereum.org>
 discussions-to: https://ethereum-magicians.org/t/eip-4938-removal-of-getnodedata/8893
-status: Stagnant
+status: Last Call
```

Comment:
```suggestion
status: Draft
```
Hi @MariusVanDerWijden thank you for authoring and continue to drive this EIP. Happy to see this EIP being revived. While looking at the history, the EIP was Draft moved to Stagnant, hence update the status to Draft to revive. Once it's revived to Draft, feel free to create PR to move to status: Review -> Last Call -> Final.

I know this process probably is a bit longer. Please let us know if this EIP is already widely adopted / implemented by client and I could try to help bring to the EIP Editor's meeting to advocate for an expedite procedure.

Reply:
Please also take a look at the linter error and try to fix all the linter errors

Reply:
It's been implemented by all execution layer clients

Reply:
It does need to go back to draft unfortunately, sorry :(

Reply:
moved to draft